clang/lib/Sema/SemaChecking.cpp

```diff
@@ -11815,6 +11815,18 @@ isArithmeticArgumentPromotion(Sema &S, const ImplicitCastExpr *ICE) {
          S.Context.getFloatingTypeOrder(From, To) < 0;
 }
 
+static clang::analyze_format_string::ArgType::MatchKind
```

Comment:
```suggestion
static analyze_format_string::ArgType::MatchKind
```

Maybe we can remove this namespace?